src/adapters/github.js

```diff
@@ -107,6 +107,16 @@ class GitHub extends PjaxAdapter {
     );
   }
 
+  // @override
+  getBranches(opts) {
+    return new Promise((resolve, reject) => {
+      this._get('/branches', opts, (err, res) => {
```

Comment:
Looks like this only gets first page only https://developer.github.com/v3/repos/branches/#list-branches, either parsing HTML or iterate through Link header to get all

Reply:
ahh, ok thanks, so now the current implementation request for first page only and since on every tree reload there's branches requesting we will have quite a lot of calls to endpoint, so storing the current retrieved branches will also be ideal, right ?

Reply:
> so now the current implementation request for the first page only.

If you mean the `current implementation` is the fetching file list on PR then Yes. However, what we're doing here is separate feature (getting branch list) so it looks like a bug (not too much critical) when the code doesn't show all branch if the amount of over 100